services/site/src/components/homepage/Card.tsx

```diff
@@ -13,9 +13,19 @@ export interface CardProps {
   difficulty: ChallengeDifficulty;
   challengeSlug: string;
   challengeNumber: number;
+  mobileFriendly?: boolean;
 }
 
-const Card: React.FunctionComponent<CardProps> = ({ className, heading, levels, finishedLevels, difficulty, challengeSlug, challengeNumber }) => {
+const Card: React.FunctionComponent<CardProps> = ({
+  className,
+  heading,
+  levels,
+  finishedLevels,
+  difficulty,
+  challengeSlug,
+  challengeNumber,
+  mobileFriendly,
```

Comment:
I would prefer `isMobileFriendly`. But you don't have to change it since we don't really have a convention here and most boolean variables don't start with a verb